src/main.rs

```diff
@@ -46,5 +46,9 @@ fn main() {
     let client_addr =
         args.flag_client_address.parse::<SocketAddr>().expect("Expect to parse client address");
 
+    println!("Starting Emerald Connector - v{}",
+             VERSION.unwrap_or("unknown"));
+    println!("Listen for connection on http://{}", addr);
+
```

Comment:
Ah, then it should log it b default. Right now you run app and wait, wait, wait, until you figure out it's already running just doesn't notify user about it